code/modules/food_and_drinks/restaurant/customers/_customer.dm

```diff
@@ -142,3 +142,38 @@
 	leave_mad_lines = list("Aye dios mio, I'm out of here.", "Esto es ridículo! I'm leaving!", "I've seen better cooking at taco campana!", "I though this was a restaurant, pero es porquería!")
 	leave_happy_lines = list("Amigo, era delicio. Thank you!", "Yo tuve el mono, and you friend? You hit the spot.", "Just the right amount of spicy!")
 	wait_for_food_lines = list("Ay ay ay, what's taking so long...", "Are you ready yet, amigo?")
+
+///MALFUNCTIONING - only shows up once per venue, very rare
+/datum/customer_data/malfunction
+	nationality = "Malfunctioning"
+	base_icon = "defect"
+	prefix_file = "strings/names/malf_prefix.txt"
+	speech_sound = 'sound/effects/clang.ogg'
+	clothing_sets = list("wires", "bad_takes")
+	orderable_objects = list(
+	/datum/venue/restaurant = list(/obj/item/toy/crayon/red = 1, /obj/item/toy/crayon/orange = 1, /obj/item/toy/crayon/yellow = 1, /obj/item/toy/crayon/green = 1, /obj/item/toy/crayon/blue = 1, /obj/item/toy/crayon/purple = 1),
+	/datum/venue/bar = list(/datum/reagent/consumable/failed_reaction = 1, /datum/reagent/spraytan = 1, /datum/reagent/reaction_agent/basic_buffer = 1, /datum/reagent/reaction_agent/acidic_buffer = 1))
+
+	found_seat_lines = list("customer_pawn.say(pick(customer_data.found_seat_lines))", "I saw your sector on the hub. What are the laws of this land?")
```

Comment:
```suggestion
	found_seat_lines = list("customer_pawn.say(pick(customer_data.found_seat_lines))", "I saw your sector on the hub. What are the laws of this land?", "The move speed here is a bit low...")
```

Reply:
*movement

Reply:
you could make it pick() between complaining that the movement speed is a bit fast and complaining that the movement speed is a bit slow

or you could be EXTRA fancy and make it pick whether it's from a fast or a slow server upon being spawned, so that it won't complain about the movement speed being too fast one line and then complain about the movement speed being too slow the next

Reply:
> make it pick whether it's from a fast or a slow server
the terry/basil check :trollface: